src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -17,6 +17,11 @@
 
 // This is a generated sample ("FlattenedAsync", "empty_response_type_with_response_handling")
 
+// sample-metadata
+//   title: Test response handling for methods that return empty
+//   description: Test response handling for methods that return empty
+//   usage: dotnet run
```

Comment:
Also, is this binary name the same in all platforms (eg on dotnet on Windows vs mono on Linux)

Reply:
Based on: https://stackoverflow.com/questions/37738106/net-core-vs-mono

`dotnet` is the binary name for dotnet core which is developed by Microsoft and works on Linux. So I think we should use that?

Reply:
Looking at https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore2x, that seems to be correct. And it's also the name on Windows, right? OK.